src/MoBi.Presentation/Presenter/EditParametersInContainerPresenter.cs

```diff
@@ -161,6 +167,11 @@ private ISelectReferenceAtParameterPresenter getNewReferencePresenterFor(IContai
          return referencePresenter;
       }
 
+      private string getContainerName(IContainer container)
+      {
+         return string.IsNullOrEmpty(container.Name) ? $"New {_typeResolver.TypeFor(container)}" : container.Name;
```

Comment:
Although we have other instances where we use strings directly in presenters like this, it's our best practice to not do so.

Can you use this `NewWindow` in `MoBi.Assets.AppConstants.Captions` for the middle option in the conditional